installer/install.iss

```diff
@@ -2758,6 +2798,8 @@ begin
     end else if RdbSSH[GS_Plink].Checked then begin
         Data:='Plink';
         RecordChoice(PreviousDataKey,'Plink Path',EdtPlink.Text);
+        if TortoisePlink.Checked then
+            RecordChoice(PreviousDataKey,'Tortoise Option','true');
```

Comment:
There is no corresponding `false` statement... Maybe this instead?

```
RecordChoice(PreviousDataKey,'Using Tortoise Plink',TortoisePlink.Checked);`
```

Reply:
I thought it was just noise to track this if the user was not requesting it.  It will not be present in any current installs.  I can always set it if you would prefer.

Reply:
The problem is: if a user will check this box, but in a subsequent install _uncheck_ it, it won't be recorded with the new preference. The old preference will prevail, forever.

Reply:
For the average install, isn't the install-options.txt re-written for each install?  If the user is installing with /saveinf == /loadinf then yes, the old preference will prevail.  Perhaps I misunderstood this.  Having said that, to avoid issues, I will change to always set the turtle option.  I was trying to avoid this because I see the use of plink options going away over time (just an opinion).

Reply:
Just noting that 'Plink Path' is not cleared if changing to Open SSH.  Similar to the tortoise check box, it will show the previous value when saveinf == loadinf.  No big deal since it does not affect execution.

Reply:
> For the average install, isn't the install-options.txt re-written for each install?

Yes, but the choices are not replayed from the `install-options.txt` file, but via InnoSetup's mechanism, from the registry.

> Just noting that 'Plink Path' is not cleared if changing to Open SSH.

Right, but that is a bit different. It means that that option is remembered when the user chooses OpenSSH, and should they want to choose Plink again, conveniently the path will be remembered.

In the issue at hand, the issue is if the user checks that check box, but then makes up their mind and re-installs, _unchecking_ the checkbox. We'd want to record this _new_ preference (unchecked) rather than leave the previous preference (checked) in place.